source/shared/cpp/ObjectModel/BaseActionElement.cpp

```diff
@@ -45,3 +48,15 @@ Json::Value BaseActionElement::SerializeToJsonValue()
     root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Id)] = GetId();
     return root;
 }
+
+Json::Value BaseActionElement::GetAdditionalProperty(std::string name)
+{
+    if (m_additionalProperties.find(name) != m_additionalProperties.end())
```

Comment:
>m_additionalProperties.find(name) [](start = 8, length = 33)

Pretty irrelevant in the grand scheme of things, but you can create a local iterator variable, and return iterator-> second if found. This means there will only be one lookup to the map instead of two.